### ISSUE TITLE
Issue logging in when password contains special characters

### ISSUE THREAD

**Issue by connorff** [GDcheeriosYT/Gentrys-Quest-Python#50]
When my password contained a `#`, I received an error logging in:
![image](https://user-images.githubusercontent.com/40175891/211641176-f2ad2979-a321-4578-a045-db47fcf78769.png)

When I create a password without special characters, I can log in fine.

**Comment by GDcheeriosYT** [comment]
I'll look into it thanks! I feel it might a problem in [website project](https://github.com/GDcheeriosYT/website-project).

**Comment by GDcheeriosYT** [comment]
I think I have an idea of what's going on here.

- You can log in using passwords that contain "#" in the web.
- You cannot log in using passwords that contain "#" in Gentry's Quest

I think the reason why is that the website `POST`S the login info as json, however Gentry's Quest uses a url `GET` request to validate the information. The request library probably takes off the # because it reads it as a fragment of the url.

I'm probably going to make it so that passwords can't contain certain symbols.

**Comment by connorff** [comment]
Is it possible to `POST` the login information instead of using `GET`? This would fix the issue and also help with security, since peoples' passwords won't be visible in your server logs.

If not, an easier way is to encode the URL parameters with [urllib.parse.quote_plus](https://docs.python.org/3/library/urllib.parse.html#urllib.parse.quote_plus).

**Comment by GDcheeriosYT** [comment]
I agree. I would prefer that people's info not be shown in there. I think it would be better to use `POST`. Is it possible to get data from `POST` or make the server send something to the client? Because when logging in the game receives the player data.
